docs/sql-guide/statements/statement-database-drop.md

```diff
@@ -0,0 +1,32 @@
+---
+title: DROP DATABASE
+layout: default
+parent: Statements
+grand_parent: SQL guide
+nav_order: 7
+---
+
+# DROP DATABASE statement
+
+Drops the database from featurebase cloud. Note this statement only pertains to FeatureBase Cloud.
+
+## Syntax
+
+```sql
+DROP DATABASE database_name
+```
+
+## Arguments
+
+| Argument | Description | Further information |
```

Comment:
Let's use the standard args table:

| Argument | Data type | Description | Required | Further information |